portfolio/src/main/webapp/script.js

```diff
@@ -13,34 +13,28 @@
 // limitations under the License.
 
 
-/** @return {string} */
-function generateRandomGreeting(){
+/**
+ * @return {string} 
+ * a random greeting
+ */
+function generateRandomGreeting() {
     //TODO use google translated to get greetings in different languages
     const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+        ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
     // Pick a random greeting.
-    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
```

Comment:
Is there a reason you changed this from const to var? I think const is normally recommended wherever it is applicable :)